graph/telemetry/istio/appender/service_entry.go

```diff
@@ -201,7 +201,7 @@ func (a ServiceEntryAppender) getServiceEntry(serviceName string, globalInfo *gr
 		// handle exact match
 		// note: this also handles wildcard-prefix cases because the destination_service_name set by istio
 		// is the matching host (e.g. *.wikipedia.com), not the rested service (e.g. de.wikipedia.com)
-		if host == serviceName {
+		if host == serviceName || host == serviceName+"."+namespace {
```

Comment:
@jshaughn Looking at the whole function, it seems that the behaviour I want for my MESH_EXTERNAL SE is actually the same as what is below defined for MESH_INTERNAL. So maybe we can refactor this a little bit.

Reply:
I don't recall this very well, how the `namespace` appears in the host?

I though the host was whatever you wrote in the `host` field of the SE, s you could have `foo.bar.whatever` and wasn't related to the namespace.
Is it because `exportTo` is set to an explicit namespace (I think i never tested this part)?

Reply:
Yes indeed, `host` is whatever we write in SE, but it doesn't have to match a service name afaict. 
But wait... I'm not even sure this is right direction. I'm trying to solve a problem for which, in theory, I don't have to create a service entry ... i'm just creating a dummy one as a workaround because else Kiali won't show my metrics... need to step back a little bit :)